<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JointCalibrationWidget</class>
 <widget class="QWidget" name="JointCalibrationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="CalibrationSelectionBox">
     <property name="title">
      <string>Select the wanted calibrations</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="1">
       <widget class="QLabel" name="numberOfCalibrationLabel">
        <property name="text">
         <string>Current number of calibrations</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="endPositionCheckBox">
        <property name="text">
         <string>End Stop Position Calibration</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="numberOfCalibrationEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="pointCalibrationCheckBox">
        <property name="text">
         <string>Point Calibration</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="endMovementCheckBox">
        <property name="text">
         <string>End Stop Movement per Step Calibration</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="motorCalibrationCheckBox">
        <property name="text">
         <string>Motor Calibration</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="innerLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="jointSelectionLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="jointSelectionLabel">
         <property name="text">
          <string>Joint Selection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="typeComboBox">
         <item>
          <property name="text">
           <string>Type</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="numberComboBox">
         <item>
          <property name="text">
           <string>Number</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="jointInfoLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="movementTypeLabel">
           <property name="text">
            <string>MovementType</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="movementTypeEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="positionLayout">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="positionLabel">
           <property name="text">
            <string>Position</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="positionLineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="rangeLabel">
           <property name="text">
            <string>Range</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="minRangeEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="maxRangeEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="actuatorLable">
           <property name="text">
            <string>Actuator type</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="actuatorTypeEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="endstopNumberLabel">
           <property name="text">
            <string># Endstops</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="endstopNumberEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="includeCheckBox">
         <property name="text">
          <string>Include</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>typeComboBox</tabstop>
  <tabstop>numberComboBox</tabstop>
  <tabstop>positionLineEdit</tabstop>
  <tabstop>minRangeEdit</tabstop>
  <tabstop>maxRangeEdit</tabstop>
  <tabstop>actuatorTypeEdit</tabstop>
  <tabstop>endstopNumberEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
